data/dockerFiles/README.md

```diff
@@ -0,0 +1,21 @@
+# Dockerfiles to create a development environment
+
+In this section you can find some Dockerfiles to build an image that will have all necessary
+dependencies to build cc-oci-runtime source code and run tests.
+
+### Prerequisite:
+
+Before building the docker image, please run `make-bundle-dir.sh` as the `rootfs` generated
+by this script will be used while building the image.
+```bash
+	$ sudo ./make-bundle-dir.sh ./rootfs
```

Comment:
Since this `README.md` exists in `data/dockerFiles/`, it's reasonable to assume that the users cwd is `data/dockerFiles/`. In which case, this command should be:

```
$ sudo ../make-bundle-dir.sh ./rootfs
```